server/.env

```
@@ -0,0 +1 @@
+MONGO_URI="YOUR_LINK" 
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)